patches/common/chromium/disable_unittests.patch

```diff
@@ -0,0 +1,14 @@
+Disable GPU config test as it fails on High Sierra
```

Comment:
It would probably be better to list all disabled tests and the reasons why they're disabled in the `.patches.yaml`.

Reply:
Sure. Makes sense. Will update.
I've also added comments above some disabled test in the patch file.